docs/orchestration/faq/upgrade_environments.md

```diff
@@ -131,20 +131,20 @@ information.
 
 `FargateTaskEnvironment` provided support for configuring details of a
 Fargate Task on a Flow. This has been replaced with
-[ECSRun](./run_configs.md#ecsrun).
+[ECSRun](../flow_config/run_configs.md#ecsrun).
 
 `ECSRun` has similar config options as `FargateTaskEnvironment`, but also
 exposes raw configuration options for both
 [registering](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.register_task_definition)
 and
 [running](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task)
 ECS tasks. There are also options for common settings (e.g.  `image`, `cpu`,
-`memory`, ...). See the examples in the [ECSRun docs](./run_configs.md#ecsrun)
+`memory`, ...). See the examples in the [ECSRun docs](../flow_config/run_configs.md#ecsrun)
 for more information.
 
 Note that use of `ECSRun` requires running an [ECS
-Agent](/orchestration/agents/ecs.md), not the deprecated [Fargate
-Agent](/orchestration/agents/fargate.md).
+Agent](../agents/ecs.md), not the deprecated [Fargate
+Agent](../agents/fargate.md).
```

Comment:
I would remove the link to Fargate Agent altogether. Even if the documentation exists, we shouldn't be sending users there in this context.

Reply:
There is also the same link to Fargate Agent in line 165, I will remove it as well.